src/pages/version/BuildVariantCard/VariantTaskGroup/index.tsx

```diff
@@ -0,0 +1,133 @@
+import styled from "@emotion/styled";
+import { useVersionAnalytics } from "analytics";
+import { GroupedTaskStatusBadge } from "components/GroupedTaskStatusBadge";
+import { wordBreakCss, StyledRouterLink } from "components/styles";
+import { getVersionRoute } from "constants/routes";
+import { mapUmbrellaStatusToQueryParam } from "constants/task";
+import { size } from "constants/tokens";
+import { StatusCount } from "gql/generated/types";
+import { useQueryParam } from "hooks/useQueryParam";
+import { PatchTasksQueryParams, TaskStatus } from "types/task";
+import { string, statuses } from "utils";
+import { arraySymmetricDifference } from "utils/array";
+
+const { groupStatusesByUmbrellaStatus } = statuses;
+const { applyStrictRegex } = string;
+
+interface VariantTaskGroupProps {
+  displayName: string;
+  statusCounts: StatusCount[];
+  variant: string;
+  versionId: string;
+}
+const VariantTaskGroup: React.VFC<VariantTaskGroupProps> = ({
+  displayName,
+  statusCounts,
+  variant,
+  versionId,
+}) => {
+  const { sendEvent } = useVersionAnalytics(versionId);
+
+  const [variantSearch] = useQueryParam<string | null>(
+    PatchTasksQueryParams.Variant,
+    undefined
+  );
+  const [sorts] = useQueryParam(PatchTasksQueryParams.Sorts, undefined);
+  const [statusSearch] = useQueryParam<string[] | null>(
+    PatchTasksQueryParams.Statuses,
+    []
+  );
+  const hasStatusFilter = statusSearch.length > 0;
+  const hasVariantFilter = variantSearch !== undefined;
+  const isVariantSelected = variantSearch === applyStrictRegex(variant);
+  const hasAnyStatusOrVariantFilters = hasVariantFilter || hasStatusFilter;
+
+  const callBack = (taskSquareStatuses: string[]) => () => {
+    sendEvent({
+      name: "Click Grouped Task Square",
+      taskSquareStatuses,
+    });
+  };
+
```

Comment:
I think it would be okay to just in-line this below; `callBack` isn't very descriptive anyways!
```suggestion
```